src/views/Dashboard/Customer/Home/GeneticData/RequestAnalysis/SuccessPage.vue

```diff
@@ -103,7 +103,7 @@ export default {
       if (analysisOrderDetail) {
         this.orderStatus = analysisOrderDetail.status
         this.geneticOrderAnalysisDetail = analysisOrderDetail
-        this.trackingId = this.geneticOrderAnalysisDetail.geneticAnalysisTrackingId
+        this.trackingId = this.geneticOrderAnalysisDetail.geneticAnalysisdTrackingId
```

Comment:
to note : there's typo in polkadot provider package. to be revised. approving for now